src/kirby/components/loading-overlay/loading-overlay.component.scss

```diff
@@ -0,0 +1,29 @@
+@import "web-imports";
+
+.overlay-wrapper {
+    height: 100%;
+    width: 100%;
+    position: relative;
+
+    .spinner-wrapper {
+        height: 100%;
+        width: 100%;
+        position: absolute;
+        z-index: z(modal-overlay);
+        opacity: 0.8;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        justify-items: center;
+        
+        &.backdrop {
+            background-color: get-color('background-color');
+        }
+
+        .spinner {
+
```

Comment:
🔫  peww pewww, die whitespace.

Reply:
Done. Can this be automated in the future?

Reply:
Yes we could write some linting rules for this.

Reply:
If we used prettier for the scss files also, this would automatically be fixed.

Reply:
Also stylelint works fine for this: https://stylelint.io/user-guide/rules